src/Sparrow/Json/JsonContextPoolBase.cs

```diff
@@ -156,12 +158,14 @@ private void DisposeOfContexts()
             }
         }
 
-        protected JsonContextPoolBase()
+        protected JsonContextPoolBase(Size? maxContextSizeToKeep)
```

Comment:
create two separate ctors to not break binary compatibility

Reply:
done